src/core/state.go

```diff
@@ -545,6 +549,39 @@ func (state *BuildState) logResult(result *BuildResult) {
 	}
 }
 
+// WaitForPreloadedSubincludes waits for the preloaded subincludes to be build and preloads them into the parser
+func (state *BuildState) WaitForPreloadedSubincludes() {
+	state.preloadComplete.Do(func() {
+		wg := sync.WaitGroup{}
+		wg.Add(len(state.Config.Parse.PreloadSubincludes))
+
+		preload := func(inc string) {
+			l, err := TryParseBuildLabel(inc, "", "")
+			if err != nil {
+				log.Fatalf("failed to preload subinclude: %v", err)
+			}
+
+			if err := state.queueTarget(l, OriginalTarget, true, true); err != nil {
```

Comment:
What's the reason for the force build?

Reply:
Was just a copy paste from elsewhere. I'm using `state.WaitForBuiltTarget()`